cmd/jag/commands/watch.go

```diff
@@ -5,20 +5,24 @@
 package commands
 
 import (
+	"bufio"
+	"bytes"
 	"context"
 	"fmt"
-	"log"
 	"os"
+	"path/filepath"
+	"strings"
+	"sync"
 
 	"github.com/fsnotify/fsnotify"
 	"github.com/spf13/cobra"
 )
 
 func WatchCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:          "watch <directory> <entrypoint>",
-		Short:        "watches <directory> for changes and runs <entrypoint> on the device every time",
-		Args:         cobra.ExactArgs(2),
+		Use:          "watch <entrypoint>",
```

Comment:
I still think could be 'run --watch' long term.

I think we should change <entrypoint> to <file> like for run.